lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+require 'active_support/core_ext/module/delegation'
+require 'active_support/core_ext/class/attribute_accessors'
```

Comment:
unfortunately better-html assumes these Rails extensions are present...

Reply:
In this case better_html should require them and also add `activesupport` as runtime dependency.

Reply:
That is indeed what will eventually happen, but that's not a reason to hold this PR

Reply:
True. Just remember that now `activesupport` is a runtime dependency of this gem so that needs to be changed in the gemspec. Also you need to add  `require 'active_support'` before those two lines since it is required to require the top level file before using any piece of the framework.

Reply:
I added `require 'active_support'` before these